src/hooks/calls/user/useUserOrganizations.js

```diff
@@ -0,0 +1,34 @@
+import useSwr from 'swr';
+import { useOrgApi } from '../../api/useOrgApi';
+import { useAuthentication } from '../user/useAuthentication';
+
+export function useUserOrganizations({ username, organizationClient, options, configuration, axios }) {
+  const { state: { token } } = useAuthentication({});
+
+  const _organizationClient = useOrgApi({organizationClient, token: token?.sha1, ...configuration, axios});
+
+  const fetchUserOrg = () => {
+    let orgList = [];
+
+    if ( username ) {
+      orgList = _organizationClient.orgListUserOrgs(username).then(({ data }) => data);
+    } else {
+      orgList = _organizationClient.orgListCurrentUserOrgs().then(({ data }) => data);
+    }
+    return orgList;
+  }
+
+  const { data: orgList, error, mutate: setUserOrganizationsList }
+    = useSwr([username], fetchUserOrg, options);
+
+  return {
+    state: {
+      orgList,
+      error,
+      isLoading: !error && !orgList,
```

Comment:
isLoading should check that: 
1. Required params to do the fetching are provided.
2. There is no error nor data (orgList) returned by SWR.

So it would be something like: 
!error && !orgList && (!!username || !!token),